lego/apps/lending/views.py

```diff
@@ -0,0 +1,28 @@
+from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+
+from lego.apps.lending.models import LendableObject
+from lego.apps.lending.serializers import (
+    LendableObjectAdminSerializer,
+    LendableObjectSerializer,
+)
+from lego.apps.permissions.api.permissions import LegoPermissions
+from lego.apps.permissions.api.views import AllowedPermissionsMixin
+from lego.apps.permissions.constants import EDIT
+
+
+class LendableObjectViewSet(AllowedPermissionsMixin, viewsets.ModelViewSet):
+    queryset = LendableObject.objects.all()
```

Comment:
Should the restriction on viewing certain objects not be reflected here or am I missing something?